src/nsmgr/internal/pkg/cmd/root.go

```diff
@@ -0,0 +1,48 @@
+// Copyright (c) 2020 Cisco and/or its affiliates.
+//
+// SPDX-License-Identifier: Apache-2.0
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at:
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// Package cmd - cobra commands for forwarder
+package cmd
+
+import (
+	"context"
+
+	"github.com/networkservicemesh/cmd-nsmgr/src/nsmgr/internal/pkg/flags"
+	"github.com/spf13/cobra"
+)
+
+func init() {
```

Comment:
Consider:
```golang
func init() {
	cmd := runCmd
	rootCmd.AddCommand(cmd)
	Flags(cmd.Flags())
	cobra.OnInitialize(flags.FromEnv(flags.EnvPrefix, flags.EnvReplacer,cmd.Flags()))
}
```

Using bits in the [flags](https://github.com/networkservicemesh/sdk/tree/master/pkg/tools/flags) package and defining your own Flags(*pflags.FlagSet) function that adds all your default flags to it.  Should allow the triming off of a bunch of utility code and simplify things.